tools/validate-schemas.ts

```diff
@@ -0,0 +1,71 @@
+import fs from 'fs-extra';
+import upath from 'upath';
+import { logger } from '../lib/logger';
+import { capitalize } from './docs/utils';
+import * as Schemas from './schemas/schema';
+
+void (async () => {
```

Comment:
i would prefer to use jest test for better output on PR's

Reply:
Okay...adding a test instead